src/geometry/GeometricalParameters.cc

```diff
@@ -53,13 +53,14 @@ GeometricalParameters::GeometricalParameters()
   fMagnetTotalSizeZ = 4*m; //updates during construction
   fTrackingStationTotalSizeZ = 2.62*m; //updates during construction
   fMagnetZPos = 40*m; //updates during construction
-  fFASER2TotalSizeZ = 10.24*m;
+  fFASER2TotalSizeZ = 25*m;//10.24*m;
```

Comment:
As mentioned elsewhere, this change requires updating the positioning in the various configuration files.